techniques/aria/ARIA24.html

```diff
@@ -2,64 +2,55 @@
 <html lang="ja">
    <head>
       <meta charset="UTF-8" />
-      <title>ARIA24: Semantically identifying a font icon with role="img"</title>
+      <title>ARIA24: アイコンフォントを意味的に識別するために、 role="img" を使用する</title>
```

Comment:
```suggestion
      <title>ARIA24: アイコンフォントを意味的に識別するために role="img" を使用する</title>
```
タイトルなので、読点は取ってしまいます。